kratos/utilities/variable_utils.h

```diff
@@ -735,11 +735,9 @@ class KRATOS_API(KRATOS_CORE) VariableUtils
     {
         KRATOS_TRY
 
-#pragma omp parallel for
-        for (int k = 0; k< static_cast<int> (rNodes.size()); ++k) {
-            NodesContainerType::iterator it_node = rNodes.begin() + k;
-            it_node->FastGetSolutionStepValue(rVariable) = rValue;
-        }
+        block_for_each(rNodes, [&](Node<3>& it_node) {
+            it_node.FastGetSolutionStepValue(rVariable) = rValue;
```

Comment:
```suggestion
        block_for_each(rNodes, [&](Node<3>& rNode) {
            rNode.FastGetSolutionStepValue(rVariable) = rValue;
```

Reply:
thats also what I wanted to suggest
See the [Style Guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide)

Reply:
:+1: